### PR TITLE
Issue26

### PR DESCRIPTION
**Pull request by drawnwren**
Fixes for the second part of Issue #26. I removed the require and added a variable declaration. I also changed the English a little bit so that the variable names reflect the names that will make the tests pass.

### CODE REVIEW

**Comment by awkay** [comment]
Sorry, you are misunderstanding the instructions. A :cards/by-id table is an entry that would be in your app state. I admit the instructions are a little hard to follow. The intention is for you to make:

```
(def cars-table 
   { :cars/by-id  { 1 {:db/id 1 ...} ... } })
```

etc.

**Comment by awkay** [comment]
I pushed up solutions to these exercises on master. Feel free to look. If you have suggestions to the exercise instructions that would be fine. I want to encourage the use of David's suggested table naming. The vars are there as pretend app state..so perhaps renaming the defs in the exercises would make it clearer.

**Comment by drawnwren** [comment]
Yep, I got the same solutions as the ones that you pushed up. I think some of the confusion was coming from the terminology itself. i.e. Table vs Database in this context. The two things I was trying to fix with my edits were:
1. The instructions didn't explicitly state to make your DB named cars-table.
2. The instructions in the second one sounded a lot like they were talking about mutable state, i.e. modifying cars-table vs creating a new table with cars-table in it.

Either way, I got to the right solutions - I'm sure other people will be able to as well. Thanks for the great resource!

**Comment by awkay** [comment]
Was there anything left here we needed to merge?
